GenericTrigger.lua

```diff
@@ -617,27 +617,27 @@ function GenericTrigger.Add(data, region)
           end
```

Comment:
A draft documentation is at: http://www.wowace.com/addons/weakauras-2/pages/custom-triggers-temporary/